Makefile

```diff
@@ -1,17 +1,20 @@
 .PHONY: build test clean prepare update
 
 GO=CGO_ENABLED=0 go
-GOFLAGS=-ldflags
 
 MICROSERVICES=cmd/device-random
 
 .PHONY: $(MICROSERVICES)
 
+VERSION=$(shell cat ./VERSION)
+
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/devoce-random.Version=$(VERSION)"
```

Comment:
This looks like a typo, "devoce" should be "device"

Reply:
@mhall119 I appreciate your review.  That's really my typo. I've corrected it

Reply:
@mhall119 please help review my change of this PR